src/app/blogs/[id]/page.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import Card from "react-bootstrap/Card";
+import Link from "next/link";
+import useSWR, { Fetcher } from "swr";
+const ViewDetailBlog = ({ params }: { params: { id: string } }) => {
+  const fetcher: Fetcher<IBlog, string> = (url: string) =>
+    fetch(url).then((res) => res.json()); //gợi ý cái blog cho mình
+
+  const { data, isLoading } = useSWR(
+    `https://652c0736d0d1df5273ef0e4e.mockapi.io/api/v1/blogs/${params.id}`,
+    fetcher,
+    {
+      revalidateIfStale: false,
+      revalidateOnFocus: false,
+      revalidateOnReconnect: false,
+    }
+  );
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div>
+      <div className="my-3">
+        <Link
+          className="btn-back"
+          href={"/blogs"}
```

Comment:
@HoVanDi - pls update this ref due to a bug happend
![image](https://github.com/hoihmyit/crud_app/assets/137279219/b9e0f397-07cb-420c-a006-ec27fb419a65)